src/index.ts

```diff
@@ -63,7 +74,13 @@ async function webtransportBiDiStreamToStream (bidiStream: any, streamId: string
   let writerClosed = false
   let readerClosed = false;
   (async function () {
-    await (writer.closed.then((ok: any) => ({ ok })).catch((err: any) => ({ err })))
+    const result: {ok: unknown} | {err: Error} = await (writer.closed.then((ok: unknown) => ({ ok })).catch((err: any) => ({ err })))
+    if ('err' in result) {
+      const msg = result.err.message
+      if (!(msg.includes('aborted by the remote server') || msg.includes('STOP_SENDING'))) {
+        log.error(`WebTransport writer closed unexpectedly: streamId=${streamId} err=${result.err.message}`)
+      }
+    }
```

Comment:
```suggestion
    const err: Error | undefined = await writer.closed.catch(err => err)
    if (err != null) {
      const msg = err.message
      if (!(msg.includes('aborted by the remote server') || msg.includes('STOP_SENDING'))) {
        log.error(`WebTransport writer closed unexpectedly: streamId=${streamId} err=${err.message}`)
      }
    }
```

Reply:
Ah TIL: this is a Promise<undefined>: https://streams.spec.whatwg.org/#default-writer-class-definition